modules/ROOT/pages/push/admin-ui.adoc

```diff
@@ -0,0 +1,92 @@
+include::{partialsdir}/attributes.adoc[]
+
+[[admin-ui]]
+== Using the Admin UI
+
+The Unified Push Server can be accessed via its RESTful endpoints, as well over an administrative user interface.
+
+=== Login and landing page
+
+Once you have the Unified Push Server running, click on it's route in OpenShift and login with your OpenShift credentials. You will see one PushApplication already created. The PushApplication is a logical construct, that represents an overall mobile application container for its _different_ mobile platforms, which are called Variants.
```

Comment:
We should mention the permissions that you need to grant when logging in with Openshift OAuth for the first time.